javascript.js

```diff
@@ -0,0 +1,67 @@
+const features = [
+  {
+    name: 'Akuma',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+  },
+  {
+    name: 'Ken Masters',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+  },
+  {
+    name: 'Samus Aran',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+  },
+  {
+    name: 'Princess Peach',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+
+  },
+  {
+    name: 'Luigi',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+
+  },
+  {
+    name: 'Mario',
+    image: './images/speaker_02.png',
+    technology: 'Emeritus Professor, Korea Advanced Institute of Science and Technology',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. At, officiis amet ab possimus impedit unde quo perspiciatis in mollitia accusamus?',
+  },
+];
+
+features.forEach((feature) => {
+  const card = `
+    <div class="speakers">
+        <img class="speaker-img" src="${feature.image}" alt="featured speaker one">
+        <div class="descriptions">
+            <h4>${feature.name}</h4>
+            <div class="tech">${feature.technology}</div>
+            <p>${feature.description}
+            </p>
+        </div>
+    </div>`;
+  document.getElementById('feature-block').innerHTML += card;
+});
+
+const hamburgericon = document.querySelector('.hamburger-button');
+const navigationBar = document.querySelector('.nav-bars');
+
+hamburgericon.addEventListener('click', () => {
```

Comment:
- [ ] It would be nice to make the mobile menu the size of the mobile version, and it should have a transparent background, or the dark color of the top nav. Kindly fix it.
- [ ] The mobile menu should cover the whole screen in mobile view unlike the current situation where it does not

Your design
![image](https://user-images.githubusercontent.com/81932478/169438082-82576656-977f-4e53-92a8-14a13a1b6642.png)

Reply:
fixed

Reply:
Implemented 👍